### PR TITLE
 CORS Config

### PR DESCRIPTION
**Pull request by smirfolio**
Make it possible to configure CORS Policy, this add to external Web Browser client to query the server

 In addition it will be a step to update the Drupal7 to Drupal8

### CODE REVIEW

**Comment by ymarcon** [comment]
What are these "optional: true" in the package-lock.json?

**Comment by smirfolio** [comment]
package-lock.json is autogenerated depending on node/npm - stuff build - version, may be need to put it on .gitignore 
For now i just edited them

File: mica-webapp/src/main/conf/application-dev.yml [hunk 1]
```diff
@@ -68,3 +68,18 @@ sets:
...
+# @Params : allowedMethods : String ("OPTIONS,GET,POST,PUT,DELETE ...", "*")
+# @Params : allowedHeaders : String ("Accept,Content-type ...", "*")
+# @Params : maxAge : Long (0 ~ 3600 etc..)
+#
+
+#cors:
+#  allowCredentials: true
+#  allowedOrigins: "http://localhost:8088"
+#  allowedMethods: "OPTIONS,GET,POST"
+#  allowedHeaders: "*"
```
Comment (ymarcon) [inline mica-webapp/src/main/conf/application-dev.yml]: maxAge default value is missing

File: mica-webapp/src/main/conf/application-dev.yml [hunk 2]
```diff
@@ -68,3 +68,18 @@ sets:
     # "<second> <minute> <hour> <day-of-month> <month> <day-of-week> <year>"
     # Current cron set to run every day at mid-night
     cron: "0 0 * * * ?"
+
+#
+# Cors Configuration
+# @Params : allowCredentials : boolean (true, false)
```
Comment (ymarcon) [inline mica-webapp/src/main/conf/application-dev.yml]: Please remove `@Params : `

**Pull request merged**